src/components/section/index.tsx

```diff
@@ -1,29 +1,52 @@
-import React, { FC, ReactNode } from 'react';
+import React, { ElementType, FC, ReactNode } from 'react';
 
-import { Box, Heading, Text } from '@chakra-ui/react';
+import { Box, chakra, useMultiStyleConfig } from '@chakra-ui/react';
 
 import Stack from '../stack';
 
 interface Props {
-  variant: 'hero';
+  variant?: 'hero' | 'regular';
   title: string;
   text?: string;
   image?: () => ReactNode;
+  as?: ElementType;
 }
 
-const Section: FC<Props> = ({ title, text, image, variant }) => {
+const Section: FC<Props> = ({
+  as = chakra.div,
```

Comment:
Why do we need to pass the `as`? Can't we always use `chakra.div`?

Reply:
I did a clean up and we don't need to pass any `as` now.